cde-core/resource/js/cdf-dd.js

```diff
@@ -1881,11 +1881,12 @@ $(function() {
     },
     content: function(data, settings, original) {
       // If it is string assume it is an array.
+      var json;
       if(String == data.constructor) {
-        eval('var json = ' + data);
+        json = JSON.parse(data);
```

Comment:
@jvelasques, I am not very experienced in JS, but isn't it better to use `stringify()`:
https://gist.github.com/akhayrutdinov/0f5c8f5c8a3c5b5c2a8e
?